### ISSUE TITLE
Crash after quickly doing two chores from chorewindow

### ISSUE THREAD

**Issue by VNRARA** [patzly/grocy-android#702]
Go to chore window and quickly one after another press the "play" button for the listed chore: crash. 

```
--------- beginning of crash
05-21 04:59:30.648 18191 18191 E AndroidRuntime: FATAL EXCEPTION: main
05-21 04:59:30.648 18191 18191 E AndroidRuntime: Process: xyz.zedler.patrick.grocy, PID: 18191
05-21 04:59:30.648 18191 18191 E AndroidRuntime: java.lang.IllegalStateException: Fragment ChoreEntryBottomSheet not attached to a context.
05-21 04:59:30.648 18191 18191 E AndroidRuntime: 	at androidx.fragment.app.Fragment.requireContext(SourceFile:18)
05-21 04:59:30.648 18191 18191 E AndroidRuntime: 	at androidx.fragment.app.Fragment.getResources(Unknown Source:0)
05-21 04:59:30.648 18191 18191 E AndroidRuntime: 	at androidx.fragment.app.Fragment.getString(SourceFile:1)
05-21 04:59:30.648 18191 18191 E AndroidRuntime: 	at xyz.zedler.patrick.grocy.helper.DownloadHelper$20.$r8$lambda$f1S-ikT8_mLI0CqAY7CZV4RjqYE(SourceFile:63)
05-21 04:59:30.648 18191 18191 E AndroidRuntime: 	at androidx.camera.view.PreviewView$1$$ExternalSyntheticLambda0.onResponse(SourceFile:109)
05-21 04:59:30.648 18191 18191 E AndroidRuntime: 	at xyz.zedler.patrick.grocy.helper.DownloadHelper$$ExternalSyntheticLambda7.onResponse(Unknown Source:9)
05-21 04:59:30.648 18191 18191 E AndroidRuntime: 	at xyz.zedler.patrick.grocy.web.CustomStringRequest$$ExternalSyntheticLambda0.onResponse(SourceFile:27)
05-21 04:59:30.648 18191 18191 E AndroidRuntime: 	at xyz.zedler.patrick.grocy.web.CustomStringRequest.deliverResponse(SourceFile:11)
05-21 04:59:30.648 18191 18191 E AndroidRuntime: 	at com.google.android.gms.common.images.zaa.run(SourceFile:210)
05-21 04:59:30.648 18191 18191 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:942)
05-21 04:59:30.648 18191 18191 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:99)
05-21 04:59:30.648 18191 18191 E AndroidRuntime: 	at android.os.Looper.loopOnce(Looper.java:226)
05-21 04:59:30.648 18191 18191 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:313)
05-21 04:59:30.648 18191 18191 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8757)
05-21 04:59:30.648 18191 18191 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
05-21 04:59:30.648 18191 18191 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
05-21 04:59:30.648 18191 18191 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1067)
--------- beginning of main
05-21 04:59:34.492 20463 20463 E r.patrick.grocy: Not starting debugger since process cannot load the jdwp agent.
```

**Comment by dominiczedler** [comment]
Is hopefully fixed with d89fdf6ea6cdd8b02259947397c1b0183dc5c9f2. Will be released today or tomorrow.

**Comment by dominiczedler** [comment]
Duplicate of #699, should be fixed with the latest version `v3.2.1`. If not, you can write here again.

**Comment by VNRARA** [comment]
My bad srry
